site/docs/hosting/vercel.md

```diff
@@ -83,6 +83,20 @@ const bot = new Bot(token);
 export default webhookCallback(bot, "http");
```

Comment:
```suggestion
export default webhookCallback(bot, "std/http");
```

Maybe try using `std/http` instead of `http`? It literally works out when I'll change `http` to `std/http`.

Reply:
@Qkfw Thank you, I'll test it soon and fix the examples if there are no problems

Reply:
Do you have an estimate on when _soon_ will be? Take your time, just don't want to see this get lost like the other pull requests